src/chains/ethereum/ethereum/tests/helpers/compare-chain-state.ts

```diff
@@ -0,0 +1,73 @@
+import { Address as EthereumJsAddress } from "ethereumjs-util";
+import { isDeepStrictEqual } from "util";
+import Blockchain from "../../src/blockchain";
+import { GanacheTrie } from "../../src/helpers/trie";
+
+/**
+ * Gets all db data from a trie.
```

Comment:
I've added a _little_ more here, but I also added a supporting comment for where this is used. Since the function just _gets_ the data and isn't explicitly used only for comparison purposes.